### PR TITLE
{tools}[GCCcore/10.3.0] num2words v0.5.10 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3702.slaking.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/f47bc99c65913dd3f855e598366c738d for a full test report.

File: easybuild/easyconfigs/n/num2words/num2words-0.5.10-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+dependencies = [
+    ('Python', '3.9.5'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('docopt', '0.6.2', {
```
Comment (boegel) [inline easybuild/easyconfigs/n/num2words/num2words-0.5.10-foss-2021a.eb:19]: `docopt` 0.6.2 is already included with `Python` 3.9.5, so there's no need to install it again?

That also implies that just using `PythonPackage` as easyblock makes more sense

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3103.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e8b31e91f0abc20803aa327f440ea90e for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3103.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/a86063a30272ff0e65e07538390543fe for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**